docs/data-import.md

```diff
@@ -11,6 +11,8 @@ Streamfield data that did not match with our new definitions had data migrations
 - Wide image was mapped to the Image streamfield
 - Aligned image was mapped to the new Image streamfield
 
+Note: we subsequently re-added a wide image block, but this is for new images only, not imported ones.
```

Comment:
**Praise**: Thank you for documenting this! I love your commitment to ensuring the docs are up to date 👍 

**Suggestion (non-blocking)**

You could use [mkdocs-material's fancy admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage), like this

```
!!! note

    we subsequently re-added a wide image block, but this is for new images only, not imported ones.
Comment
```

Reply:
Our prettier set-up doesn't allow saving the file with the indentation, so I have left this one.